src/VCR/Configuration.php

```diff
@@ -121,16 +121,26 @@ class Configuration
     private $blackList = array('src/VCR/LibraryHooks/', 'src/VCR/Util/SoapClient', 'tests/VCR/Filter');
 
     /**
-     * The mode which determines how requests are handled
-     *
-     * Currently supported modes:
-     *      - new_episodes (Always allows new HTTP requests)
-     *      - once (Will allow new HTTP requests the first time the cassette is created then throw an exception after that)
-     *      - none (Will never allow new HTTP requests)
+     * Always allow to do HTTP requests and add to the cassette. Default mode.
+     */
+    const MODE_NEW_EPISODES = 'new_episodes';
```

Comment:
Good idea! 

I'm a bit torn where to put the constants. Either on the configuration or directly on the `VCR` class. 

I wanted to provide a nicer interface for adding [events](https://github.com/php-vcr/php-vcr/blob/master/src/VCR/VCREvents.php) as well and was thinking about putting it all on the `VCR` object. That would look like this:

```
VCR::configuration()
    ->setMode(VCR::MODE_NEW_EPISODES)
    ->on(VCR::BEFORE_RECORD, function() {...})
```

What would you like more as a user? `Configuration` or `VCR`. I'm more in favour of `VCR` right now because using only a single interface seems kind of neat.